src/python/grpcio_tests/tests_aio/unit/server_test.py

```diff
@@ -579,15 +582,18 @@ async def test_maximum_concurrent_rpcs(self):
         # Build the channel
         channel = aio.insecure_channel(bind_address)
         # Deplete the concurrent quota with 3 times of max RPCs
-        rpcs = []
+        rpc_tasks = []
         for _ in range(3 * _MAXIMUM_CONCURRENT_RPCS):
-            rpcs.append(channel.unary_unary(_BLOCK_BRIEFLY)(_REQUEST))
```

Comment:
Can you explain the change here in a bit more detail? `channel.unary_unary(_BLOCK_BRIEFLY)(_REQUEST)` should return a `UnaryUnaryCall`, [which is an Awaitable](https://github.com/grpc/grpc/blob/01907a776771eae2b2b5082fcf6d4ca2bd8386ce/src/python/grpcio/grpc/aio/_base_call.py#L138). So [`asyncio.wait`](https://docs.python.org/3/library/asyncio-task.html#asyncio.wait) should be able to accept these. Is the issue that the return type of `asyncio.wait` does not match up with the parameters expected by `self.loop.create_task`?

Reply:
`asyncio.wait` does not accept `UnaryUnaryCall` anymore. This code works fine before 3.11 but will start getting this error in 3.11:
```
Traceback (most recent call last):
  File "/usr/local/google/home/xuanwn/.pyenv/versions/311env/lib/python3.11/site-packages/grpc/aio/_call.py", line 576, in _invoke
    serialized_response = await self._cython_call.unary_unary(
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "src/python/grpcio/grpc/_cython/_cygrpc/aio/call.pyx.pxi", line 331, in unary_unary
  File "src/python/grpcio/grpc/_cython/_cygrpc/aio/call.pyx.pxi", line 149, in grpc._cython.cygrpc._AioCall._set_status
  File "/usr/local/google/home/xuanwn/.pyenv/versions/3.11.0/lib/python3.11/asyncio/tasks.py", line 515, in _on_completion
    f.exception() is not None)):
    ^^^^^^^^^^^
AttributeError: 'UnaryUnaryCall' object has no attribute 'exception'
```

likely because [passing coroutine objects to asyncio.wait() directly is forbidden starting 3.11](https://docs.python.org/3/library/asyncio-task.html#asyncio.wait).

In order to get it to work, we need create a task, and since `asyncio.create_task` only accept coroutine, we need a wrapper to wrap `UnaryUnaryCall`.

Reference:
* [Awaitable objects in Python after 3.10](https://stackoverflow.com/questions/76566490/awaitable-objects-in-python-after-3-10)

Reply:
I see. Python version-specific. Makes sense. Increases the urgency for multiversion Bazel tests.